roles/openshift_version/tasks/main.yml

```diff
@@ -191,6 +191,7 @@
     when:
     - not is_containerized | bool
     - openshift_release is defined
+    - not 'oo_etcd_to_migrate' in groups
```

Comment:
This looks like it will always be populated and always be present.

Reply:
Oops, I was thinking this group is created only during migration.

Now migrate playbook would set `etcd_migration_in_progress` fact instead